src/main/scala/com/netflix/edda/aws/AwsCrawlers.scala

```diff
@@ -1057,8 +1057,20 @@ class AwsReservedInstanceCrawler(val name: String, val ctx: AwsCrawler.Context)
 class AwsReservedInstancesOfferingCrawler(val name: String, val ctx: AwsCrawler.Context) extends Crawler {
   val request = new DescribeReservedInstancesOfferingsRequest
 
-  override def doCrawl()(implicit req: RequestId) = backoffRequest { ctx.awsClient.ec2.describeReservedInstancesOfferings(request).getReservedInstancesOfferings }.asScala.map(
-    item => Record(item.getReservedInstancesOfferingId, ctx.beanMapper(item))).toSeq
+  override def doCrawl()(implicit req: RequestId) = {
+    val it = new AwsIterator() {
+      def next() = {
+        val response = backoffRequest { ctx.awsClient.ec2.describeReservedInstancesOfferings(request.withNextToken(this.nextToken.get)) }
+        this.nextToken = Option(response.getNextToken)
+        response.getReservedInstancesOfferings.asScala.map(
+          item => {
+            Record(item.getReservedInstancesOfferingId, ctx.beanMapper(item))
+          }).toList
+      }
+    }
+    val list = it.toList.flatten
+    list
```

Comment:
This could be simplified to:

```
it.toList.flatten
```

Reply:
Yep I thought about it but @copperlight was quicker in accepting the PR. I can open another one with the improvement if you want

Reply:
No worries, I'll just update it directly.